services/provider/server/server.go

```diff
@@ -278,7 +278,7 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 		Name: monConfigMap,
 		Kind: "ConfigMap",
 		Data: mustMarshal(map[string]string{
-			"data":     cmData[0], // Address of first mon
+			"data":     strings.Join(cmData, ","), // IP Address of all mon's
```

Comment:
No need to join information that was split just a few lines above
```suggestion
			"data":    configmap.Data, // IP Address of all mon's
```

You can also remove the split above and change the condition to check empty string instead of len

Reply:
Done